synapse/models/infotech.py

```diff
@@ -301,6 +301,16 @@ def getModelDefs(self):
                 ('it:app:snort:hit', ('guid', {}), {
                     'doc': 'An instance of a snort rule hit.',
                 }),
+                ('it:reveng:function', ('guid', {}), {
+                    'doc': 'A hash identifying a function',
```

Comment:
Lets not call this a "hash" ( even though in our case it is ).  Maybe "A function within an executable." or something.

Reply:
:+1: